tests/CacheCompat/CommonCache.Test.Unit/CacheExecutionTests.cs

```diff
@@ -22,10 +22,17 @@ private static async Task<LabUserData> GetPublicAadUserDataAsync()
             var api = new LabServiceApi();
             LabResponse labResponse = (await api.GetLabResponseFromApiAsync(UserQuery.PublicAadUserQuery).ConfigureAwait(false));
             return new LabUserData(
-                labResponse.User.Upn, 
-                labResponse.User.GetOrFetchPassword(), 
-                labResponse.User.AppId,
-                labResponse.User.TenantId);
+              labResponse.User.Upn,
+              labResponse.User.GetOrFetchPassword(),
+              labResponse.User.AppId,
+              labResponse.User.TenantId);
+
+            //return new LabUserData(
+            //    "user",
+            //    "",
+            //    "1d18b3b0-251b-4714-a02a-9956cec86c2d",
+            //    "49f548d0-12b7-4169-a390-bb5304d24462");
```

Comment:
Should be removed?

Reply:
Lab was down and this helped, so i decided it leave it